code/script.js

```diff
@@ -1,9 +1,39 @@
-// A function that adds and remove the class "active" on the section you click on.
+/*// A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
 function toggle() {
   this.classList.toggle("active")
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
+Selects an HTML element, and calls a function which will be executed when the element is clicked.
 document.getElementById("section1").onclick = toggle
+*/
+
+
+const btn = document.getElementById("menu_button");
+
+function toggleDrinkMenu() {
+    this
+        .classList
+        .toggle('open')
+}
+btn.onclick = toggleDrinkMenu;
+
+// script for accordion
+
+var acc = document.getElementsByClassName("dropacc");
```

Comment:
Try to use `const` or `let` before using `var` if possible when you are declaring the variables!
```suggestion
const acc = document.getElementsByClassName("dropacc");
```